docs/application/dotnet/guides/nui/overview.md

```diff
@@ -20,6 +20,22 @@ NUI offers the following main features:
 -   Easy-to-use [animation framework](animation.md) which hides the complexity of the underlying three-dimensional math
 -   Keyboard, touch, and mouse handling
 
+Some reference applications:
+<table style="text-align:center;">
+  <tr>
+    <th><img src="https://github.com/dalihub/nui-demo/raw/master/wearable-samples/res/wapps.png"/></th>
```

Comment:
@mijin85cho, these images are placed on an external Github repo, is it okay to link to an image outside?

Reply:
The image is in https://github.com/dalihub/nui-demo/, and the repository seems to be managed by the people in Samsung in charge of graphics/DALi/NUI. So I think it is okay.